### ISSUE TITLE
Error in getting DynamicWedges...

### ISSUE THREAD

**Issue by GLN** [cuthbertLab/music21#289]
I'm encountering a converter.parse() error on the MusicXML file for this work: 

[https://musescore.com/user/6105546/scores/2843936](https://musescore.com/user/6105546/scores/2843936) (GitHub won't allow me to attach a .mxl file) 

The error is "Error in geting DynamicWedges...Measure no. 268 P14" (sic) and out of 13 randomly selected MusicXML files on MuseScore, I'm seeing this same error on 5 of them. 

I'm using 5.1.0 (congrats) but this error is present in version 4 as well. 

music21 is an important part of an app I've built so I'm hoping you're willing to fix.

**Comment by mscuthbert** [comment]
If you can trim and isolate the score to the smallest possible .xml fragment that recreates the error then the core dev team can figure out what the problem is.  Otherwise, it will probably need to wait until the error occurs on a smaller file -- a complete Mahler symphony movement is a lot to debug.

**Comment by GLN** [comment]
OK, I've isolated a two-measure snippet of a single part that should illustrate the problem. It's attached as a .txt file -- just change the file extension to .xml and feed it to converter.parse().

[Issue #289.txt](https://github.com/cuthbertLab/music21/files/1850028/Issue.289.txt)

The issue seems to be a "number" attribute mismatch between the wedge pairs -- here's the sequence of wedge elements in the file: 

`<wedge type="diminuendo" number="6"/>`
`<wedge type="crescendo" number="0"/>`
`<wedge type="stop" number="6"/>`
`<wedge type="stop" number="6"/>`

If I change the crescendo's "number" attribute from "0" to "6" then there is no error.

Clearly this is a problem with data derived from erroneous notation (the crescendo starts before the diminuendo finishes) but if music21 is able to handle it I'd be very grateful.

**Comment by deepio** [comment]
I tend to have this error when exporting musicxml from `MuseScore 3.2.3`. 

These wedges broke for me. 
```xml
<measure number="3" width="482.12">
      <direction placement="below">
        <direction-type>
          <wedge type="diminuendo" number="3" default-y="-75.00"/>
          </direction-type>
        <staff>1</staff>
        </direction>
      <note>
        <rest>
          <display-step>C</display-step>
          <display-octave>4</display-octave>
          </rest>
        <duration>12</duration>
        <voice>1</voice>
        <type>quarter</type>
        <staff>1</staff>
        </note>
...
         <wedge type="stop" number="3" relative-x="-436.18"/>
...
         <wedge type="stop" number="3" relative-x="-124.62"/>
```

**Comment by deepio** [comment]
@mscuthbert Any update on this issue? It's still present in Music21 `5.7.2` and MuseScore `3.3.4`. A spanner is created without any other object and crashes. I think a warning that the file is invalid would be better, but it does not need to crash.

https://github.com/cuthbertLab/music21/blob/ed66817fdbb4430f9de10dd63c32d088eb01ab0b/music21/musicxml/xmlToM21.py#L3607-L3608

Returns an empty list. 

https://github.com/cuthbertLab/music21/blob/ed66817fdbb4430f9de10dd63c32d088eb01ab0b/music21/musicxml/xmlToM21.py#L3609-L3614
Crash

Here is a fleshed out file from MuseScore.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE score-partwise PUBLIC "-//Recordare//DTD MusicXML 3.0 Partwise//EN" "http://www.musicxml.org/dtds/partwise.dtd">
<score-partwise>
  <identification>
    <encoding>
      <software>MuseScore 3.3.4</software>
      </encoding>
    </identification>
  <part-list>
    <score-part id="P1">
      <part-name>Clarinet</part-name>
      <score-instrument id="P1-I1">
        </score-instrument>
      </score-part>
    </part-list>
  <part id="P1">
    <measure number="1" width="279.65">
      <note default-x="85.75" default-y="-585.00">
        <pitch>
          <step>A</step>
          <octave>3</octave>
          </pitch>
        <voice>1</voice>
        <type>half</type>
        </note>
      <direction placement="below">
        <direction-type>
          <wedge type="stop" number="0"/>
          </direction-type>
        </direction>
      </measure>
    </part>
  </score-partwise>
```

**Comment by jacobtylerwalls** [comment]
fixed in c9c258a

**Comment by mscuthbert** [comment]
Thanks Jacob!
